lib/image_view.dart

```diff
@@ -107,137 +148,176 @@ class _ImageviewState extends State<Imageview> {
+                              },
+                              child: Column(
+                                mainAxisAlignment:
+                                    MainAxisAlignment.spaceEvenly,
+                                children: const <Widget>[
+                                  Icon(
+                                    Icons.filter,
+                                    color: Colors.white,
+                                  ),
+                                  Text(
+                                    "Filter",
+                                    style: TextStyle(color: Colors.white),
+                                  ),
+                                ],
+                              ),
                             ),
-                            Text(
-                              "Next",
-                              style: TextStyle(color: Colors.white),
+                            TextButton(
+                              onPressed: () {
+                                if (files.isNotEmpty) {
+                                  widget.list.imagelist.add(files[index]);
+                                  widget.list.imagepath.add(files[index].path);
+                                } else {
+                                  widget.list.imagelist.add(widget.file);
+                                  widget.list.imagepath.add(widget.file.path);
+                                }
+                                Navigator.of(context).pushNamed(
+                                  RouteConstants.multiDelete,
+                                  arguments: widget.list,
+                                );
+                              },
+                              child: Column(
+                                mainAxisAlignment:
+                                    MainAxisAlignment.spaceEvenly,
+                                children: const <Widget>[
+                                  Icon(
+                                    Icons.arrow_forward,
+                                    color: Colors.white,
+                                  ),
+                                  Text(
+                                    "Next",
+                                    style: TextStyle(color: Colors.white),
+                                  ),
+                                ],
+                              ),
                             ),
                           ],
                         ),
                       ),
-                    ],
+                    ),
                   ),
-                ),
+                ],
               ),
             ),
-          ],
-        ),
-      ),
     );
   }
 
-  void goToFilterImageScreen({File file}) {
-    Navigator.of(context).pushNamed(
-      RouteConstants.filterImageScreen,
-      arguments: {
-        'file': file,
-        'list': widget.list,
-      },
-    );
-  }
+  // void goToFilterImageScreen({File file, @required BuildContext context}) {
```

Comment:
remove these lines. I don't think they are needed anymore.

Reply:
Done the changes. You can review it now